pontoon-to-webext.js

```diff
@@ -0,0 +1,164 @@
+#! /usr/bin/env node
+
+/* eslint-disable promise/avoid-new */
+
+const propertiesParser = require('properties-parser');
+const path = require('path');
+const FS = require('q-io/fs');
+const argv = require('minimist')(process.argv.slice(2));
+
+const Habitat = require('habitat');
```

Comment:
My understanding of Habitat is that it loads stuff from a `.env` file into `process.env`, so it's not exactly used directly… Having said that, the code comes from `pontoon-to-json`, so I'ld kind of like to leave it in, if that's okay.  😉